spec/requests/assignments_spec.rb

```diff
@@ -8,6 +8,10 @@
   let(:course) { create(:course) }
   let!(:assignment) { create(:assignment, course_id: course.id) }
 
+  before(:each) do
```

Comment:
why not create the user in `api_helper.rb` and remove all the before blocks in each spec?